src/index.js

```diff
@@ -24,6 +29,13 @@ sgf('ACM', function (err, results) {
         const linter = linters[key]
         const fileList = filePaths.filter(minimatch.filter(key, { matchBase: true }))
         if (fileList.length) {
+            // If current working directory is not the git root -> resolve file paths accordingly
+            if (sgf.cwd !== process.cwd()) {
+                var relpath = path.relative(process.cwd(), sgf.cwd) + '/'
+                for (var i in fileList) {
```

Comment:
This won't work on Windows I guess. Let's remove `/` and just use `path.resolve` that accepts `Array<string>`.